src/azul/plugins/repository/tdr_anvil/__init__.py

```diff
@@ -636,71 +795,6 @@ def convert_column(value):
         }
     }
 
-
-class TDRAnvilBundle(TDRBundle):
-    entity_type: EntityType = 'biosample'
-
-    def add_entity(self,
-                   entity: EntityReference,
-                   version: str,
-                   row: MutableJSON
-                   ) -> None:
-        pk_column = entity.entity_type + '_id'
-        self._add_entity(
-            manifest_entry={
-                'uuid': entity.entity_id,
-                'version': version,
-                'name': f'{entity.entity_type}_{row[pk_column]}',
-                'indexed': True,
-                'crc32': '',
-                'sha256': '',
-                **(
-                    {'drs_path': self._parse_drs_uri(row.get('file_ref'))}
-                    if entity.entity_type == 'file' else {}
-                )
-            },
-            metadata=row
-        )
-
-    def add_links(self,
-                  bundle_fqid: BundleFQID,
-                  links: Links,
-                  entities_by_key: Mapping[KeyReference, EntityReference]) -> None:
-        def link_sort_key(link: JSON):
-            return link['activity'] or '', link['inputs'], link['outputs']
-
-        def key_ref_to_entity_ref(key_ref: KeyReference) -> str:
-            return str(entities_by_key[key_ref])
-
-        self._add_entity(
-            manifest_entry={
-                'uuid': bundle_fqid.uuid,
-                'version': bundle_fqid.version,
-                'name': 'links',
-                'indexed': True
-            },
-            metadata=sorted((
-                {
-                    'inputs': sorted(map(key_ref_to_entity_ref, link.inputs)),
-                    'activity': None if link.activity is None else key_ref_to_entity_ref(link.activity),
-                    'outputs': sorted(map(key_ref_to_entity_ref, link.outputs))
-                }
-                for link in links
-            ), key=link_sort_key)
-        )
-
-    def _add_entity(self,
-                    *,
-                    manifest_entry: MutableJSON,
-                    metadata: AnyMutableJSON
-                    ) -> None:
-        name = manifest_entry['name']
-        assert name not in self.metadata_files, name
-        self.manifest.append(manifest_entry)
-        self.metadata_files[name] = metadata
-
-    def _parse_drs_uri(self, file_ref: Optional[str]) -> Optional[str]:
-        if file_ref is None:
-            return None
-        else:
-            return self._parse_drs_path(file_ref)
+    def _columns(self, entity_type: EntityType) -> set[str]:
```

Comment:
This seems out of place at the bottom of the file. Could it be put either above or below `_convert_column`?